infrastructure/datastore/db.go

```diff
@@ -0,0 +1,110 @@
+package datastore
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"taskmanager/domain/model"
```

Comment:
Just a small detail: Imports tend to follow this order by convention, each section separated by a newline.
```
1. Native packages.
2. External packages.
3. Internal packages.
```

[Here](https://stackoverflow.com/a/38714480/4521547)'s an example. The `goimports` tool can help you to do this automatically.

Reply:
Right now my VS Code setup is doing that arrangement automatically when I save, I will check that `goimports` tool to see if the arrangement is different, thanks 😄